Tests/AppConfig/config.yml

```diff
@@ -29,4 +29,4 @@ liip_functional_test:
 services:
     faker.provider.foo:
         class: Liip\FunctionalTestBundle\Tests\AppConfig\DataFixtures\Faker\Provider\FooProvider
-        tags: [ { name: hautelook_alice.faker.provider } ]
```

Comment:
the doc needs to be updated accordingly

Reply:
See my todos.

Reply:
@theofidry I removed the doc part concerning the faker provider. This part was only telling about hautelook installation which is not needed anymore.